src/main/scala/io/iohk/ethereum/rlp/package.scala

```diff
@@ -20,10 +20,14 @@ package object rlp {
 
   def encode[T](input: T)(implicit enc: RLPEncoder[T]): Array[Byte] = RLP.encode(enc.encode(input))
 
+  def encodeToRlp[T](input: T)(implicit enc: RLPEncoder[T]): RLPEncodeable = enc.encode(input)
```

Comment:
it is also used in PV63.scala but renamed to `rlpEncode`
for example https://github.com/input-output-hk/etc-client/pull/19/files#diff-048743a755e4bf01956ef907e6704010R90